src/Raven.Studio/typescript/viewmodels/database/indexes/deleteIndexesConfirm.ts

```diff
@@ -3,35 +3,85 @@ import dialog = require("plugins/dialog");
 import database = require("models/resources/database");
 import dialogViewModelBase = require("viewmodels/dialogViewModelBase");
 import messagePublisher = require("common/messagePublisher");
+import index = require("models/database/index/index");
+import indexDefinition = require("models/database/index/indexDefinition");
 
-class deleteIndexesConfirm extends dialogViewModelBase {
+class indexInfoForDelete {
+    indexName: string;
```

Comment:
can this class have 2 static factory methods? One for index and second for index definition?

Reply:
done :)